b2/console_tool.py

```diff
@@ -314,6 +314,41 @@ def run(self, args):
         return 0
 
 
+class CreateKey(Command):
+    """
+    b2 create-key [--duration <validDurationSeconds>] [--bucket <bucketId>] [--prefix <namePrefix>]
```

Comment:
Later we can add a `--json` option to print the response JSON, instead of condensing to just the keyId and key.

Reply:
You've mentioned this before but I dont see --json being used anywhere in the project. 
I dont think it would be hard to add, we could just take it in as an arg and then whenever 
a class Command is called check for an option --json flag then just use json.dumps.